historyserver/dashboard/homepage/index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
```

Comment:
I think we can alias a new page (like ip:8080/historyserver) to index page. So the user can go back to history server index page and clear the cookies, so user can select another cluster.  Currently, ip:8080 (with cookies) will be redirected to overview page of dashboard instead of homepage of historyserver.